Source/ViewModels/TPStreamPlayerViewModel.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  TPStreamPlayerViewModel.swift
+//  TPStreamsSDK
+//
+//  Created by Testpress on 14/06/24.
+//
+
+import Foundation
+import Combine
+
+@available(iOS 13.0, *)
```

Comment:
- Extracted the TPStreamPlayerView into view model extends observable object as we can't modify the Swift UI state on callback.